webpack.config.js

```diff
@@ -19,5 +20,14 @@ module.exports = {
         }
       }
     ]
-  }
+  },
+  plugins: [
+    new CompressionPlugin({
+      filename: '[path].gz[query]',
+      algorithm: 'gzip',
+      test: /\.(js|css|html)$/,
+      threshold: 8192,
```

Comment:
You may want to reduce the threshold to zero - you're adding `.gz` to all `.js` files, so any `.js` file under 8kb will return a 404 error.

Reply:
oh I didn't know that. I was under the impression that you shouldn't set the threshold too low. thanks for letting me know